cfme/control/explorer/alerts.py

```diff
@@ -124,6 +126,71 @@ def is_displayed(self):
         )
 
 
+class MonitorAlertsAllView(BaseLoggedInPage):
+
+    result_header = Text('h5.ng-binding')
+    alerts_list = AllAlertsList('list-view-pf-view ng-isolate-scope')
```

Comment:
ng-isolate-scope shouldn't be included here, and I think it would be a good idea to pass this as kwarg for readability of what locator component you're passing (is that a title? name? id?)